docs/pages/indexer.mdx

```diff
@@ -195,8 +195,8 @@ Block number: 2042
 Tables: schema,Hooks,StoreMetadata,NamespaceOwner,InstalledModules,ResourceAccess,Systems,FunctionSelector,SystemHooks,SystemRegistry,ResourceType,Counter
 Information about Counter
 {
-  id: '0x5FbDB2315678afecb367f032d93F642f64180aa3____Counter',
-  address: '0x5FbDB2315678afecb367f032d93F642f64180aa3',
+  id: '0x0x6e9474e9c83676b9a71133ff96db43e7aa0a4342____Counter',
+  address: '0x0x6e9474e9c83676b9a71133ff96db43e7aa0a4342',
```

Comment:
there seem to be too many `0x` here, I assume that's unintentional?

Reply:
Yes, sorry. Thank you for catching it!